tcms/telemetry/urls/telemetry_urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from tcms.telemetry.views import TestCaseBreakdownView
+
+urlpatterns = [
+    url(r'^testcase-breakdown', TestCaseBreakdownView.as_view(), name='testcase-breakdown')
+]
```

Comment:
Why not just urls.py ?

Reply:
Because this is how it is done in other packages:
```
+ --testcase
|    +-- urls
|     |     +-- __init__.py # just imports of the other two files
|     |     +-- case_urls.py
|     |     +-- cases_urls.py
+ --testplans
|    +-- urls
|     |     +-- __init__.py # just imports of the other two files
|     |     +-- plan_urls.py
|     |     +-- plans_urls.py
```

Reply:
But `testruns/urls.py` - this is the Django way, the others are legacy.